meetings/2021-07/july-13.md

```diff
@@ -0,0 +1,512 @@
+
+CM: Yeah just two just to share that in the previous proposal that was a getter on the input of the realm. That get a give you access to the global object that was removed. There's nothing that prevents of adding that in the future. If we feel that that's really what is needed. Just that at the moment, we believe that's a foot Khan and it's very error-prone. And yes, they are previous around that, like, the iPhones and the DM toes come with without issue the identity. This Tissue. 
+
+DE: so, I'm kind of having trouble following some of the discussion about use cases because I get to their use cases for shared objects are almost but they're also use cases. Many use cases presented for Caldwell boundary problems and the existence of one class of use cases doesn't negate that the other class. So I could I can see the argument for for adding this a good proposal but I don't see why this gets tied up in that other one. This realms proposal meets, the use cases that the champion group has the champion that, you know, presented the motivation and brought it to stage two. So, this, you know, the current the current path makes sense to me, even if it doesn't everybody's problem. 
+
+JHD: So to be cleared an the at the time it was brought to stage two. It's solved the shared object use cases also, so callable boundary around is effectively a brand new proposal that happens to solve some of the use cases of the proposal that went to stage two. So it's I think it's an it's not really fair to characterize this as like, oh well, these are just two overlapping proposals because, like, essentially the stage 2 proposal was kind of replaced very recently replaced with a One that does not solve all of the use cases that the stage 2 proposal solved. The fact that the champion group at only has a subset of those use cases, doesn't also negate that the stage 2 presumption. We all had was that all of those use cases were solved and that's what my next sort of queue item is around. 
```

Comment:
```suggestion
JHD: So to be clear, at the time it was brought to stage 2, it solved the shared object use cases also, so callable boundary is effectively a brand new proposal that happens to solve some of the use cases of the proposal that went to stage 2. So it's I think it's not really fair to characterize this as like, "oh well, these are just two overlapping proposals" because essentially the stage 2 proposal was kind of replaced  - very recently - with one that does not solve all of the use cases that the stage 2 proposal solved. The fact that the champion group only has a subset of those use cases, doesn't also negate that the stage 2 presumption we all had was that all of those use cases were solved and that's what my next sort of queue item is about. 
```